File: DAL/Models/Sales/SalesOrderQuoteApproverList.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DAL.Models.Sales
{
    public class SalesOrderQuoteApproverList
    {
        [Key]
        public long? SalesOrderQuoteApproverListId { get; set; }
        public long SalesOrderQuoteId { get; set; }
        public long EmployeeId { get; set; }
        public int Level { get; set; }
        public int StatusId { get; set; }
        public int MasterCompanyId { get; set; }
        public string CreatedBy { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
    }
}

File: QuickApp.Pro/ClientApp/app/models/sales/SalesOrderQuoteApproverList.ts
import { ISalesOrderQuoteApproverList } from "./ISalesOrderQuoteApproverList";

export class SalesOrderQuoteApproverList implements ISalesOrderQuoteApproverList {
    salesOrderQuoteApproverListId: number;
    salesOrderQuoteId: number;
    employeeId: number;
    firstName: string;
    employeeCode: string;
    email: string;
    level: number;
    statusId: number;
    masterCompanyId: number;
    createdBy: string;
    updatedBy: string;
    createdDate: number;
    updatedDate: number;
    
}